tools/tests/com.liferay.ide.xml.search.ui.tests/src/com/liferay/ide/xml/search/ui/tests/LiferayHookXmlTests.java

```diff
@@ -368,9 +371,34 @@ protected void testServiceTypeAndServiceImplHyperlink()
     {
     }
 
-    // TODO
-    protected void testServiceTypeAndServiceImplValidation()
-    {
+    @Test
+    protected void testServiceTypeAndServiceImplValidation() throws Exception
+    {	
+    	//IDE-1810
+    	if( shouldSkipBundleTests() ) return;
```

Comment:
Check the source formatting in this method, everything should be aligned on the left hand side.  YOu can use our liferay-ide formatter preferences found in the repository under /liferay-ide/resources.  You can import that source formatter profile.